packages/eslint-plugin-wdio/package.json

```
@@ -17,7 +17,8 @@
         "require": "./cjs/index.js"
       },
       "./cjs/index.js"
-    ]
+    ],
+    "./package.json": "./package.json"
```

As I mentioned we don't need it here
```suggestion
    ]
```

---

ack, updated!